react/SlideToggle/SlideToggle.js

```diff
@@ -0,0 +1,72 @@
+import styles from './SlideToggle.less';
+import React, { Component, PropTypes } from 'react';
+import TickIcon from '../TickIcon/TickIcon';
+import classnames from 'classnames';
+
+const LEFT = 'left';
+const RIGHT = 'right';
+
+export default class SlideToggle extends Component {
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    text: PropTypes.string.isRequired,
+    onToggle: PropTypes.func,
+    position: PropTypes.oneOf([LEFT, RIGHT]),
+    hideLabel: PropTypes.bool,
+    children: PropTypes.node,
+    checked: PropTypes.bool
+  }
+
+  static defaultProps = {
+    position: RIGHT,
+    hideLabel: false
+  };
+
+  handleChange = event => {
+    const { onToggle } = this.props;
+    if (onToggle) {
+      onToggle(event.target.checked);
+    }
+  }
+
+  renderLabel = () => {
+    const { position, text } = this.props;
+    return (
+      <span
```

Comment:
We should probably use the `FieldLabel` component here so it corresponds to the text and label standards